src/components/lesson-metadata-modal.js

```diff
@@ -156,6 +206,22 @@ function LessonMetadataModal({ lesson, mode, isVisible, onSave, onClose }) {
             showTime={{ defaultValue: moment(`${firstEnabledHour}:00`, 'HH:mm'), format: 'HH:mm', hideDisabledOptions: true }}
             />
         </FormItem>
+        <FormItem name="enableSequence" valuePropName="checked" hidden>
+          <Checkbox />
+        </FormItem>
+        {mode === LESSON_MODAL_MODE.create && (
+          <Collapse className="LessonMetadataModal-sequenceCollapse" activeKey={isSequenceExpanded ? ['sequence'] : []} onChange={handleRepetitionsCollapseChange} ghost>
+            <CollapsePanel header={t('createSequence')} key="sequence" forceRender>
+              <Alert className="LessonMetadataModal-sequenceInfo" message={t('sequenceInfoBoxHeader')} description={t('sequenceInfoBoxDescription')} type="info" showIcon />
+              <FormItem label={t('sequenceIntervall')} name="sequenceIntervall" rules={[]}>
```

Comment:
Do `rules` need to be provided at all if there is no validation?

Reply:
No, deleted it here, but I actually provided some rules now for the `sequenceCount` field.